nextflow_schema.json

```diff
@@ -81,7 +81,14 @@
                 },
                 "orf_caller": {
                     "type": "string",
-                    "description": "Specify which ORF caller you would like to run."
+                    "description": "possible alternatives: prodigal, default: prodigal.",
+                    "default": "prodigal"
+                },
+                "training_file": {
+                    "type": "string",
+                    "default": "-t $params.prodigal_trainingfile",
+                    "hidden": true,
+                    "description": "add a training file to prodigal"
                 }
```

Comment:
This chunk should define `prodigal_trainingfile`. No need for a default, the user should specify a path if he/she wants to use one, otherwise this is not set. (The default is hence null.)